Jenkinsfile

```diff
@@ -649,7 +649,20 @@ try {
   }
 
   stage('Integration Test') {
-    parallel 'Python GPU': {
+	parallel 'Python CPU': {
+      node('mxnetlinux-cpu') {
+        ws('workspace/it-python-cpu') {
+          init_git()
+          unpack_lib('cpu')
+          timeout(time: max_time, unit: 'MINUTES') {
+            sh "${docker_run} cpu --dockerbinary docker PYTHONPATH=./python/ python tests/python/integrationtest/onnx_backend_test.py"
+            sh "${docker_run} cpu --dockerbinary docker PYTHONPATH=./python/ python tests/python/integrationtest/test_layers.py"
```

Comment:
Are these two not running nosetest?

Reply:
test_layers is running nosetest and I will change the command, but onnx_backend_test is not using nosetest

Reply:
If they're running nosetests, then that's the wrong command to execute them.

For non-nose-tests please create a parallel directory to tests/python because all tests in tests/python are nosetests.

Reply:
onnx_backend_test uses pytest. 

I will shift the whole folder from "tests/python/integrationtest" to "tests/integrationtests". I will run all onnx related tests using pytest. And make corresponding changes in Jenkinsfile. Will that be alright?

Reply:
please create a folder called ``tests/python-pytest/onnx`` instead. Right, then you'll have to make the appropriate changes to the dockerfiles